orchagent/nexthopkey.h

```diff
@@ -115,7 +117,8 @@ struct NextHopKey
     {
         if (srv6_nh)
         {
-            return ip_address.to_string() + NH_DELIMITER + srv6_segment + NH_DELIMITER + srv6_source;
+            return ip_address.to_string() + NH_DELIMITER + srv6_segment + NH_DELIMITER + srv6_source + NH_DELIMITER 
+                   + srv6_vpn_sid + NH_DELIMITER;
```

Comment:
@yuezhoujk @shuaishang Do we need NH_DELIMITER after VPN SID at the end?

Reply:
We'd better have. Since `srv6_vpn_sid` could be empty and we limit the length of tokenized keys should be 4

```C++
NextHopKey(const std::string &str, bool overlay_nh, bool srv6_nh = false)
    {
        if (str.find(NHG_DELIMITER) != string::npos)
        {
            std::string err = "Error converting " + str + " to NextHop";
            throw std::invalid_argument(err);
        }
        if (srv6_nh == true)
        {
            vni = 0;
            weight = 0;
            srv6_bfd_disc = 0;
            auto keys = tokenize(str, NH_DELIMITER);
            if (keys.size() != 4)
            {
                std::string err = "Error converting " + str + " to srv6 Nexthop";
                throw std::invalid_argument(err);
            }
```